### PR TITLE
Create Flatpak manifest

### PR DESCRIPTION
**Pull request by danrabbit**
It works yay!

### CODE REVIEW

**Comment by Marukesu** [comment]
maybe because the baseapp doesn't install the defaults-settings, so it don't have AccountsServices in the sandbox? the same apply to the elementary sdk

**Comment by Marukesu** [comment]
also, it's need the `rename-icon` tag

**Comment by Marukesu** [comment]
adding `--system-talk-name=org.freedesktop.Accounts` and `--talk-name=org.gnome.Shell.Screenshot` make it run, but conceal won't work.

**Comment by danrabbit** [comment]
Hey @Marukesu thanks for the review! The icon issue should be resolved and conceal is now done over the dbus interface. I'm stilling having an issue running though:

`Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown`

Any ideas?

File: io.elementary.screenshot.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+app-id: io.elementary.screenshot
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+base: io.elementary.BaseApp
+base-version: juno-19.08
```
Comment (bilelmoussaoui) [inline io.elementary.screenshot.yml:5]: There's a 20.08 version based on fdo 20.08/ gnome 3.38

**Comment by Marukesu** [comment]
The error is the flatpak app-id (`io.elementary.screenshot`) don't match the app name (`io.elementary.screenshot-tool`), neither the gsettings schema (so it don't migrate the app settings), the solution is keep the app-id to `io.elementary.screenshot-tool` until #178 is merged. Also, the manifest need to add `--filesystem=home` and `--socket=pulseaudio` for save, conceal and sound.

**Comment by fabiozaramella** [comment]
#178 has just been merged so you may want to update this PR once again 😅

**Comment by danrabbit** [comment]
@fabiozaramella updated!

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -24,6 +24,20 @@ jobs:
...
+  flatpak:
+    runs-on: ubuntu-latest
+
+    container:
+      image: docker.io/bilelmoussaoui/flatpak-github-actions
+      options: --privileged
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: bilelmoussaoui/flatpak-github-actions@v1
```
Comment (Marukesu) [inline .github/workflows/main.yml:36]:
```suggestion
    - uses: bilelmoussaoui/flatpak-github-actions@v2
```

File: io.elementary.screenshot.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  - '--socket=pulseaudio'
+
+  - '--filesystem=home'
+  - '--talk-name=org.gnome.Shell.Screenshot'
+
+  - '--metadata=X-DConf=migrate-path=/io/elementary/screenshot/'
+modules:
+  - name: handy
+    buildsystem: meson
+    sources:
```
Comment (Marukesu) [inline io.elementary.screenshot.yml:26]:
```suggestion
    config-opts:
      - '-Dexamples=false'
      - '-Dtests=false'
    sources:
```

File: io.elementary.screenshot.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/libhandy.git
+        tag: '1.0.0'
```
Comment (Marukesu) [inline io.elementary.screenshot.yml:29]:
```suggestion
        tag: '1.0.1'
```

**Review by Marukesu — CHANGES_REQUESTED** [review]
some minor changes.

**Comment by Marukesu** [comment]
also, if we wanna to update the runtime to 3.38 and the base to juno-20.08 we need to build canberra in the manifest, similar to camera here:
https://github.com/elementary/camera/blob/6e3e8e5751a8f884502242d5b969c53a85b36d5c/io.elementary.camera.yml#L27

**Comment by danrabbit** [comment]
@Marukesu thanks for catching those!

**Review by Marukesu — APPROVED** [review]

**Pull request merged**